src/Pim/index.tsx

```diff
@@ -247,6 +247,7 @@ const CollectionRoutes = withStyles(styles)(withRouter(
               if (this.itemUndefined(itemUid)) {
                 return PageNotFound();
               }
+              const isBirthday = itemUid.match(/^birthdays\|*/) !== null;
```

Comment:
I think this is a bit hacky, but more than that, it's unnecessary.
It can just be done in SyncGate. Just make the selector also depend on this setting and only generate it if the setting is set. If you make the journal read-only the no-editing will already be handled for you (if it doesn't that's a bug in etesync-web that can be fixed).
This will make it so you don't even need to touch this file.

Reply:
Done, but the editing buttons are still enabled.

Reply:
I just checked, and the code indeed doesn't deal with readOnly. I'll fix that in a moment.